src/main/java/org/apache/mesos/hdfs/state/PersistentState.java

```diff
@@ -1,23 +1,28 @@
 package org.apache.mesos.hdfs.state;
 
 import com.google.protobuf.InvalidProtocolBufferException;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
 import org.apache.mesos.MesosNativeLibrary;
+import org.apache.mesos.Protos;
 import org.apache.mesos.Protos.FrameworkID;
 import org.apache.mesos.hdfs.config.SchedulerConf;
+import org.apache.mesos.hdfs.util.HDFSConstants;
 import org.apache.mesos.state.Variable;
 import org.apache.mesos.state.ZooKeeperState;
 
 import java.io.*;
-import java.util.HashSet;
-import java.util.Set;
+import java.util.HashMap;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.TimeUnit;
 
 public class PersistentState {
   private static String FRAMEWORK_ID_KEY = "frameworkId";
-  private static String NODES_KEY = "nodes";
-  private static String NAMENODE_KEY = "namenode";
+  private static String NAMENODES_KEY = "nameNodes";
+  private static String JOURNALNODES_KEY = "journalNodes";
+  private static String DATANODES_KEY = "dataNodes";
   private ZooKeeperState zkState;
+  public static final Log log = LogFactory.getLog(PersistentState.class);
```

Comment:
Could we move this constant to the top? public static then private static then private variables would be a good order.